src/utils/util.js

```diff
@@ -0,0 +1,18 @@
+function lcns( str ) {
```

Comment:
Please rename as we should avoid abbreviations unless they're very well understood, from the function name you should know what it is doing.